timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -246,21 +252,25 @@ limitations under the License.
                   close
                   close-icon="mdi-close"
                   @click:close="removeChip(chip)"
+                  @click="copyChipValue(chip)"
                   v-bind="attrs"
                   v-on="onTooltip"
                 >
-                  <v-icon v-if="chip.value === '__ts_star'" left small color="amber">mdi-star</v-icon>
-                  <v-icon v-if="chip.value === '__ts_comment'" left small>mdi-comment-multiple-outline</v-icon>
-                  <v-icon v-if="getQuickTag(chip.value)" left small :color="getQuickTag(chip.value).color">{{
-                    getQuickTag(chip.value).label
-                  }}</v-icon>
-                  <span v-if="chip.operator === 'must_not'" class="filter-chip-truncate">
-                    <span style="color: red">NOT </span>
-                    {{ (chip.field ? `${chip.field} : ${chip.value}` : chip.value) | formatLabelText }}
-                  </span>
-                  <span v-else class="filter-chip-truncate">
-                    {{ (chip.field ? `${chip.field} : ${chip.value}` : chip.value) | formatLabelText }}
-                  </span>
+                <span v-if="copiedChip === chip">Copied!</span>
```

Comment:
Instead of replacing the content of the chip, we are using the snackbar for this info. Please have a look on how this is solved with other places that we copy content and adjust accordingly. For example: https://github.com/google/timesketch/blob/master/timesketch/frontend-ng/src/components/Explore/EventDetail.vue#L344